includes/type/object/class-collection-stats-type.php

```diff
@@ -0,0 +1,168 @@
+							return ! empty( $source['min_price'] ) ? $source['min_price'] : null;
+						}
+					],
+					'maxPrice' => [
+						'type'        => 'Float',
+						'description' => __( 'Maximum price', 'wp-graphql-woocommerce' ),
+						'resolve'     => function( $source ) {
+							return ! empty( $source['max_price'] ) ? $source['max_price'] : null;
+						}
+					],
+				],
+			]
+		);
+
+		register_graphql_object_type (
+			'AttributeCount',
+			[
+				'eagerlyLoadType' => true,
+				'description' => __( 'Attribute count', 'wp-graphql-woocommerce' ),
+				'fields'      => [
+					'termId' => [
+						'type'        => [ 'non_null' => 'ID' ],
+						'description' => __( 'Term ID', 'wp-graphql-woocommerce' ),
+					],
+					'count' => [
+						'type'        => 'Int',
+						'description' => __( 'Filtered Term Count', 'wp-graphql-woocommerce' ),
+					],
+				],
+				'connections' => [
+					'term'     => [
+						'toType'         => 'TermNode',
+						'oneToOne'       => true,
+						'resolve'        => static function ( $source, $args, $context, $info ) {
+							$term_id  = ! empty ( $source->termId ) ? $source->termId : 0;
+							$taxonomy = ! empty ( $source->taxonomy ) ? $source->taxonomy : null;
+							$resolver = new TermObjectConnectionResolver( $source, $args, $context, $info, $taxonomy );
+
+							return $resolver->one_to_one()
+								->set_query_arg( 'include', [ $term_id ] )
+								->get_connection();
+						},
+					],
+				]
+			]
+		);
+
+		register_graphql_object_type(
+			'RatingCount',
+			[
+				'eagerlyLoadType' => true,
+				'description' => __( 'Rating count', 'wp-graphql-woocommerce' ),
+				'fields'      => [
+					'rating' => [
+						'type'        => [ 'non_null' => 'Int' ],
+						'description' => __( 'Rating', 'wp-graphql-woocommerce' ),
+					],
+					'count' => [
+						'type'        => 'Int',
+						'description' => __( 'Filtered Rating Count', 'wp-graphql-woocommerce' ),
+					],
+				],
+			]
+		);
+
+		register_graphql_object_type(
+			'CollectionStats',
+			[
+				'description' => __( '', 'wp-graphql-woocommerce' ),
+				'fields'      => [
+					'priceRange' => [
+						'type'        => 'PriceRangeStats',
+						'description' => __( 'Price range', 'wp-graphql-woocommerce' ),
+						'resolve'     => function( $source ) {
+							$min_price = ! empty( $source['min_price'] ) ? $source['min_price'] : null;
+							$max_price = ! empty( $source['max_price'] ) ? $source['max_price'] : null;
+							return compact( 'min_price', 'max_price' );
+						}
+					],
+					'attributeCounts' => [
+						'type'        => [ 'list_of' => 'AttributeCount' ],
+						'args'        => [
+							'page' => [
+								'type'        => 'Int',
+								'description' => __( 'Page of results to return.', 'wp-graphql-woocommerce' ),
+							],
+							'perPage' => [
+								'type'        => 'Int',
+								'description' => __( 'Number of results to return per page.', 'wp-graphql-woocommerce' ),
```

Comment:
I think we should return all the results. I don't see any type of pagination in the Store API implementation.

Reply:
Because this result list can get pretty monstrous and it would be pretty foolishly to not do this. And REST is already caching the full result list so might as well take advantage of that. If you want the full list don't set any arguments, not seeing an issue here🤷🏿‍♂️